src/Appwrite/Resque/Worker.php

```diff
@@ -2,10 +2,17 @@
 
 namespace Appwrite\Resque;
 
+use Utopia\App;
+use Utopia\CLI\Console;
+use Utopia\Logger\Log;
+use function get_class;
+
 abstract class Worker
```

Comment:
Would be great to add doc blocks for all the abstract classes for future devs.

Reply:
I documented Worker class with PHPDoc, hopefully, that's what you requested. Is that okay? https://github.com/appwrite/appwrite/commit/fd1dcae24709251695243d2a3bc31fb2fae288d6

Reply:
Perfect 👍